llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -953,7 +953,7 @@ def FeatureStdExtSstc
     : RISCVExtension<"sstc", 1, 0,
                      "'Sstc' (Supervisor-mode timer interrupts)">;
 
-def FeaturesStdExtSsqosid
+def FeatureStdExtSsqosid
```

Comment:
For exmaple, this typo can be fixed in another PR.

Reply:
Removed, thanks